test/performance/shmem_perf_suite/shmem_bw_nb_put_perf.c

```diff
@@ -35,15 +35,19 @@
 
 #include <bw_common.h>
 
+#include <pthread.h>
```

Comment:
do these includes need to be in this file? I'm not seeing where this dependency came from. I'd rather keep the thread specific elements separated out the way you have with OpenMP.